buildSrc/src/main/java/Dependencies.kt

```diff
@@ -30,7 +30,7 @@ private object Versions {
 
     const val mozilla_fxa = "0.10.0"
     const val mozilla_sync_logins = "0.10.0"
-    const val mozilla_places = "0.10.0"
+    const val mozilla_places = "0.11.0"
```

Comment:
nit: Those version numbers are currently listed in the changelog too

So this is one of the cases where we just want to bump one of them?

Reply:
I am about to merge the FxA changes which will bump all of them to 0.11. I will send a separate PR to update the changelog.